code/game/objects/items/devices/radio/headset.dm

```diff
@@ -299,3 +299,8 @@
 		secure_radio_connections[ch_name] = add_radio(src, radiochannels[ch_name])
 
 	return
+
+/obj/item/device/radio/headset/AltClick(mob/user)
+	if (command)
+		use_command = !use_command
+		to_chat(user, "<span class='notice'>You toggle high-volume mode.</span>")
```

Comment:
You could add an on/off to the message(`[use_command ? "on" : "off"]`) so it has a clearer feedback for the user.